movie-showcase/src/routes/managing/+page.svelte

```diff
@@ -0,0 +1,124 @@
+<script lang="ts">
+    import { movieArray } from "../../stores";
+    import { slideDuration } from "../../stores";
+    import { carouselItemCount } from "../../stores";
+
+    var Title: string;
+    let Year: string;
+
+    async function addToArray() {
+        const response = await fetch("/managing", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                title: Title,
+                year: Year,
+            }),
+        });
+
+        const movie = await response.json();
+
+        movieArray.update((arr) => [
+            ...arr,
+            {
+                Title: movie.Title,
+                Year: movie.Year,
+                Plot: movie.Plot,
+                Poster: movie.Poster,
+            },
+        ]);
+    }
+    function removeMovie(movie) {
+        movieArray.update((arr) => {
+            return arr.filter((m) => m !== movie);
+        });
+    }
+</script>
+
+<div class="card">
+    <!-- Submit to add a movie -->
+    <p class="title">Add a movie</p>
+    <form on:submit={addToArray}>
+        Title: <input type="text" bind:value={Title} />
+        Year: <input type="text" bind:value={Year} />
+        <input type="submit" value="Add" />
+    </form>
+
+    <!-- Movie Preview -->
+    <!-- Loop over fetched movies and show as item-->
+    <p class="title">Current Movies</p>
+    <div class="movielist">
+        {#each $movieArray as item}
+            <div class="movieitem">
+                <p>{item.Title}</p>
+                <p>{item.Year}</p>
+                <button on:click={(event) => removeMovie(item)}>remove</button>
+            </div>
+        {/each}
+    </div>
+</div>
+
+<div class="card">
+    <p class="title">Settings</p>
+    <form class="form">
+        <div class="settingOption">
+            <p>Slide time:</p>
+            <input type="number" bind:value={$slideDuration} /> Second(s)
+        </div>
+        <div class="settingOption">
+            <p>Amount of movies to show</p>
+            <input type="number" bind:value={$carouselItemCount} />
+        </div>
+    </form>
+</div>
+
+<style>
+    .card {
```

Comment:
The card is declared again here. You could move it to a component for re-usability.